src/StarterKits/LicenseManager.php

```diff
@@ -73,7 +73,7 @@ public function expireLicense()
 
         Http::post(self::OUTPOST_ENDPOINT.'expire', [
```

Comment:
Just a quick thing to note - the `Http` stuff isn't included in Laravel 6 (which [Statamic currently supports](https://github.com/statamic/cms/blob/3.1/composer.json#L16)). It's available in Laravel 7 onwards.

Although, I guess it's very unlikely the SK license stuff will get run on a L6 install anyway so maybe this doesn't even matter?

Reply:
👍  Yep, L7+ is required to install starter kits... https://github.com/statamic/cms/blob/starter-kits-2/src/Console/Commands/StarterKitInstall.php#L43-L45

Reply:
Gotcha! Sorry, I didn't read anything other than a little diff of changes 😅

Reply:
All good!